### ISSUE TITLE
Found something weird.

### ISSUE THREAD

**Issue by Daargajedan** [iam4x/pokemongo-webspoof#275]
Hey Guys,

Found something interesting, 

If i run the pokemongo-webspoof.app in an folder where the name contains a space like:
/Users/<userfolder/Desktop/pokemongo-webspoof-darwin-x64 2/pokemongo-webspoof.app

The app won't start. if i put it on my desktop everything works fine:
/Users/<userfolder/Desktop/pokemongo-webspoof-darwin-x64 2/pokemongo-webspoof.app

**Comment by flogarko** [comment]
Yeah, i have the same problem. it was in a folder called "pokemon go", it didn't work but all was fine after i rename it in "pokemongo"
